cookbooks/swift/files/default/etc/prometheus/statsd_mapping.yaml

```diff
@@ -0,0 +1,445 @@
+mappings:
+      ## Swift statsd -> statsd exporter
+      ### account/container/container auditor
+      - match: ^(internal|subrequest)*(\.)*(account|container|object)-auditor(-[0-9]*)?\.timing
+        match_type: regex
+        name: "${3}_auditor_timing"
+        labels:
+          hostname: "$2$1"
```

Comment:
This was all heavily lifted from a similar config we have running in prod; it could probably use a bit of cleanup. The **big** idea, though, is to mostly _get rid of it entirely_ by letting the application code indicate its own labelling scheme.